app/views/devise/shared/_links.html.erb

```diff
@@ -0,0 +1,25 @@
+<%- if controller_name != "sessions" %>
```

Comment:
dấu `-` ở sau thẻ nhúng code Ruby để làm gì thế em?

Reply:
Nó giống như cú pháp <% %> thôi ạ. gem devise nó generate ra thế thì em dùng thôi ạ

Reply:
sai bét, tìm hiểu lại đi nhé -_-

Reply:
Cú pháp này xóa hết khoảng trắng trước thẻ ở dòng hiện tại nếu phía trước thẻ chỉ toàn khoảng trắng